app/views/shop/_messages.html.haml

```diff
@@ -10,7 +10,7 @@
              register: ('<a auth="signup">' + t('.register') + '</a>').html_safe}
         - else
           = t '.require_customer_html',
-            {contact: ('<a href="##contact">' + t('.contact') + '</a>').html_safe,
+            {contact: (link_to t('.contact'), "#/contact", method: :get),
```

Comment:
I'm not familiar enough with Angular to know if there's a better way but happy to go with this as is or with @stveep's suggestion as it resolves the main issue which is there being a broken link.